packages/contracts/contracts/interpreters/CoinTransferETHInterpreter.sol

```diff
@@ -8,7 +8,7 @@ import "../libs/LibOutcome.sol";
  * This file is excluded from ethlint/solium because of this issue:
  * https://github.com/duaraghav8/Ethlint/issues/261
  */
-contract ETHInterpreter is Interpreter {
+contract CoinTransferETHInterpreter is Interpreter {
```

Comment:
Should use a better name...suggest `ETHOfCoinTransferInterpreter`

Reply:
TBH I think `CoinTransferETHInterpreter` is a better name than `ETHOfCoinTransferInterpreter`